randovania/games/dread/json_data/Artaria.txt

```diff
@@ -1065,7 +1071,9 @@ Extra - asset_id: collision_camera_010
   * Extra - right_shield_entity: {EMPTY}
   * Extra - right_shield_def: None
   > Door to EMMI Zone Exit Northwest
-      Space Jump or Simple IBJ
+      Any of the following:
+          Space Jump or Speed Booster or Simple IBJ
+          Flash Shift and Walljump (Intermediate)
```

Comment:
I do not believe this is possible with just Flash due to the ledge that sticks out